### ISSUE TITLE
Custom language file

### ISSUE THREAD

**Issue by evolvewebhost** [centralnicgroup-opensource/rtldev-middleware-whmcs#281]
Custom language file does not appear to be working. I have added the new language json file to /lang/overrides and updated the version in the theme.json file. For me, it only works by updating /resources/cnic/templates/cnicdomainsearch/client_theme/languages/english.json
I have also cleared WHMCS template cache several times. Php 8.1 and WHMCS 8.10.1

**Comment by AsifNawaz-cnic** [comment]
Hi @evolvewebhost,

Sorry to hear you're having trouble with custom language files in our domain search addon. Could you please let us know which version of the CNIC WHMCS Bundle you're using? This will help us look into the issue more effectively.

Thanks!

**Comment by evolvewebhost** [comment]
@AsifNawaz-cnic 23.12.0

**Comment by KaiSchwarz-cnic** [comment]
Thanks for addressing @evolvewebhost / Doug. Internal Reference: RSRMID-2078

**Comment by AsifNawaz-cnic** [comment]
Hi @evolvewebhost,

I hope you're doing well, and thank you for addressing the issue.

Initially, the custom language feature was designed for unsupported languages in our search engine addon. However, we've now extended this functionality to all languages, including officially supported ones, to ensure that changes won't be lost in future updates.

We have released version [23.13.0](https://github.com/centralnicgroup-opensource/rtldev-middleware-whmcs/releases/tag/v23.13.0) of our WHMCS Bundle. To update, simply replace the files located in this folder: `/resources/cnic/assets/js/cnicdomainsearch/client_app`.
